6_visualize/src/combine_storm_frames.R

```diff
@@ -1,9 +1,10 @@
-combine_storm_frames <- function(gif_file, task_names=NULL) {
+combine_storm_frames <- function(gif_file, animation_cfg=8, task_names=NULL) {
```

Comment:
Looks like `animation_cfg` is expected to be a list w/ `$frame_delay_cs`

Reply:
good point - default no longer works